nilrt_snac/_configs/_auditd_config.py

```diff
@@ -0,0 +1,146 @@
+import argparse
+import subprocess
+import re
+import grp
+import os
+import stat
+from nilrt_snac._configs._base_config import _BaseConfig
+from nilrt_snac._configs._config_file import _ConfigFile
+from typing import List
+from nilrt_snac import logger
+from nilrt_snac.opkg import opkg_helper
```

Comment:
Style. For consistency, could you break these into separate groups; first, the standard module imports, then the project imports? And could you alpha-sort the lines within each group?